docs/source/programming_resources/tutorial_specific/android_studio/fork_and_clone_github_repository/Fork-and-Clone-From-GitHub.rst

```diff
@@ -1,5 +1,5 @@
-Fork and Clone from GitHub 
-==========================
+Fork and Clone from GitHub :bdg-success:`AS`
+=============================================
```

Comment:
Let's make headings tags equal to the content above them. I know the specification says "at least as long" but we have thus far been really good about "exactly the length".